package.json

```diff
@@ -2,55 +2,48 @@
+    "e2e:open": "yarn --cwd ./projects/storefrontapp-e2e-cypress run cy:open",
+    "e2e:run": "yarn --cwd ./projects/storefrontapp-e2e-cypress run cy:run",
+    "e2e:run:ci": "yarn --cwd ./projects/storefrontapp-e2e-cypress run cy:run:ci",
+    "e2e:run:ci:b2b": "yarn --cwd ./projects/storefrontapp-e2e-cypress run cy:run:ci:b2b",
+    "e2e:run:ci:flaky": "yarn --cwd ./projects/storefrontapp-e2e-cypress run cy:run:ci:flaky",
+    "e2e:run:ci:cds": "yarn --cwd ./projects/storefrontapp-e2e-cypress run cy:run:ci:cds",
+    "e2e:start:open": "start-server-and-test start http-get://localhost:4200 e2e:open",
+    "e2e:start:run:ci": "start-server-and-test start:prod http-get://localhost:4200 e2e:run:ci",
+    "e2e:start:run:ci:b2b": "start-server-and-test start:prod http-get://localhost:4200 e2e:run:ci:b2b",
+    "e2e:start:run:ci:cds": "start-server-and-test start:prod http-get://localhost:4200 e2e:run:ci:cds",
+    "e2e:start:run:ci:flaky": "start-server-and-test start:prod  http-get://localhost:4200 e2e:run:ci:flaky",
+    "e2e:start:open:cds": "start-server-and-test start:prod http-get://localhost:4200 e2e:open",
+    "e2e:start:run:cds": "start-server-and-test start:prod http-get://localhost:4200 e2e:run",
     "generate:changelog": "ts-node ./scripts/changelog.ts",
     "generate:docs": "npx @compodoc/compodoc -p tsconfig.compodoc.json && ./scripts/zip-docs.sh",
     "generate:publish:docs": "yarn generate:docs && yarn publish:docs",
-    "lint": "ng lint",
     "i18n-lint": "i18n-lint -t \"{{,}}\" projects/storefrontlib/src/**/*.html feature-libs/**/*.html --exclude **/node_modules/**/*.html",
+    "lint": "ng lint",
+    "lint:styles": "stylelint \"{projects,feature-libs}/**/*.scss\"",
     "prettier": "prettier --config ./.prettierrc --list-different \"{projects,feature-libs,core-libs,integration-libs}/**/*{.ts,.js,.json,.scss,.html}\"",
     "prettier:fix": "prettier --config ./.prettierrc --list-different \"{projects,feature-libs,core-libs,integration-libs}/**/*{.ts,.js,.json,.scss,.html}\" --write",
-    "lint:styles": "stylelint \"{projects,feature-libs}/**/*.scss\"",
-    "start": "ng serve",
-    "start:ci:1905": "cross-env SPARTACUS_BASE_URL=https://spartacus-legacy.eastus.cloudapp.azure.com:9002 SPARTACUS_API_PREFIX=/rest/v2/ yarn start:prod",
-    "start:ci:2005": "cross-env SPARTACUS_BASE_URL=https://spartacus-devci767.eastus.cloudapp.azure.com:9002 SPARTACUS_API_PREFIX=/occ/v2/ yarn start:prod",
-    "build:ci:2005": "cross-env SPARTACUS_BASE_URL=https://spartacus-devci767.eastus.cloudapp.azure.com:9002 SPARTACUS_API_PREFIX=/occ/v2/ yarn build --prod",
-    "start:ci:ccv2": "cross-env SPARTACUS_BASE_URL=https://spartacus-demo.eastus.cloudapp.azure.com:8443 yarn start:prod",
-    "start:prod": "ng serve --prod",
-    "start:ssl": "ng serve --ssl",
+    "start": "env-cmd -e ci:b2c,$SPA_ENV ng serve",
+    "start:prod": "env-cmd -e ci:b2c,$SPA_ENV ng serve --prod",
+    "start:ssl": "env-cmd -e ci:b2c,$SPA_ENV ng serve --ssl",
```

Comment:
Can we drop this one? I don't remember ever using it
```suggestion
```

Reply:
Yes. Dropped.